service/src/test/java/org/cbioportal/service/impl/StructuralVariantServiceImplTest.java

```diff
@@ -54,12 +54,13 @@ public void getStructuralVariants() throws Exception {
         List<String> molecularProfileIds = Collections.singletonList("study_structural_variants");
         List<Integer> entrezGeneIds = Collections.singletonList(ENTREZ_GENE_ID_1);
         List<String> sampleIds = Collections.singletonList(SAMPLE_ID1);
+        List<StructuralVariantIdentifier> noStructVars = new ArrayList<>();
```

Comment:
Deze naam is niet verhelderend in een methode die heet `fetchStructuralVariantsNoSampleIdentifiers` :) Plz rename...

Reply:
De methode heet toch `getStructuralVariants`, of wat bedoel je precies?

Reply:
maar misschien is het duidelijker met `noStructVarIds?`?